src/resources/AnalyticsAdmin/Properties/Properties.ts

```diff
@@ -0,0 +1,72 @@
+import Resource from '../../Resource.js';
+import API from '../../../APICore.js';
+import {PageModel, Paginated} from '../../BaseInterfaces.js';
+import {PropertiesResponseMessage, PropertyModel} from './PropertiesInterfaces.js';
+
+export default class Properties extends Resource {
+    static baseUrl = `/rest/organizations/${API.orgPlaceholder}/analyticsadmin/v1`;
+    /**
+     * Build the request path, handling the optional `org` query parameter.
+     *
+     * @param route The path part of the request.
+     * @param queryParams Optional query parameters object.
+     * If this object contains an `org` property, it will override the value from the configuration.
+     * @returns The request path including formatted query parameters.
+     */
+    protected buildPathWithOrg(route: string, queryParams?: any): string {
+        return super.buildPath(route, {org: this.api.organizationId, ...queryParams});
+    }
+
+    /**
+     * List all properties
+     *
+     * @returns Promise<PageModel<PropertyModel>>
+     */
+    listProperties(pagination?: Paginated): Promise<PageModel<PropertyModel>> {
```

Comment:
We usually avoid repeating the ressource name in all the methods, it avoids repetition

`Platform.properties.list()` instead of `Platform.properties.listProperties()`

```suggestion
    list(pagination?: Paginated): Promise<PageModel<PropertyModel>> {
```

(same comment for all other methods)